packages/react/src/components/Header/HeaderAction/HeaderActionPanel.jsx

```diff
@@ -46,7 +54,13 @@ const HeaderActionPanel = ({ item, index, onToggleExpansion, isExpanded, focusRe
         onClick={() => onToggleExpansion()}
         ref={focusRef}
       >
-        {item.btnContent}
+        {inOverflow ? (
+          item.label
+        ) : isExpanded ? (
+          <Close16 fill="white" description="close-icon" />
```

Comment:
Same comments as for previous close icon

Reply:
same as other.